vscode/src/extension.ts

```diff
@@ -103,6 +108,16 @@ function registerDocumentUpdateHandlers(languageService: ILanguageService) {
   const subscriptions = [];
   subscriptions.push(
     vscode.workspace.onDidOpenTextDocument((document) => {
+      const documentType = isQsharpDocument(document)
```

Comment:
Doesn't this fire for every document opened (not just Q# or Jupyter documents)? (CC @minestarks). Do we really want to send an event for every document opened with a type of 'Other'?